invokeai/backend/model_management/model_manager.py

```diff
@@ -512,14 +515,15 @@ def offload_model(self, model_name: str) -> None:
         print(f">> Offloading {model_name} to CPU")
         model = self.models[model_name]["model"]
         model.offload_all()
+        self.current_model = None
 
         gc.collect()
         if self._has_cuda():
             torch.cuda.empty_cache()
 
     def scan_model(self, model_name, checkpoint):
         """
-        Apply picklescanner to the indicated checkpoint and issue a warning
+v        Apply picklescanner to the indicated checkpoint and issue a warning
```

Comment:
Typo?

Reply:
Thanks. Typo indeed. I think it was caught and fixed in PR #3011 but will double check.